Userland/Utilities/ps.cpp

```diff
@@ -50,10 +50,12 @@ int main(int argc, char** argv)
 
     bool every_process_flag = false;
     bool full_format_flag = false;
+    String pidlist;
 
     Core::ArgsParser args_parser;
     args_parser.add_option(every_process_flag, "Show every process", nullptr, 'e');
     args_parser.add_option(full_format_flag, "Full format", nullptr, 'f');
+    args_parser.add_option(pidlist, "Select processes by process ID", nullptr, 'q', "pidlist");
```

Comment:
This doesn't hint at the fact that they have to be split with commas, could we hint to that in there somehow?